components/tidb_query_vec_expr/src/impl_compare.rs

```diff
@@ -883,8 +889,8 @@ mod tests {
     }
 
     #[test]
-    fn test_greatest_int() {
-        let cases = vec![
+    fn test_greatest() {
+        let _int_cases = vec![
```

Comment:
What does the leading underscore mean?

Reply:
rust-analyzer told me the internal variable‘s name should start with prefix _

Reply:
I guess that warning appeared because you didn't use the variable at that time...
The variable starting with `_` is allowed not to be used.